content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -213,7 +213,7 @@ view UserInfo {
 
 ```prisma highlight=2-5;normal
 view UserInfo {
-  id    String @unique
+  id    String  @id @default(auto()) @map("_id") @db.ObjectId
```

Comment:
Validation error if you do not have `_id` with all the other stuff it needs for MongoDB?
That could possibly be an issue for us we can fix later in the `views` preview feature.
Can you create that @sabinadams ? Thanks.

Reply:
<img width="1512" alt="CleanShot 2023-02-03 at 09 45 20@2x" src="https://user-images.githubusercontent.com/18456526/216671730-c5faba1b-3de5-4d12-893b-cadc29f81802.png">

Yep, validation errors. I'll create the issue 👍🏻

Reply:
https://github.com/prisma/prisma/issues/17733